### ISSUE TITLE
Input files for the code

### ISSUE THREAD

**Issue by soummyaah** [1049451037/GCN-Align#5]
Hi,

I am extremely interested in reproducing the code from your paper. However, the initial datasets found at [JAPE](https://github.com/nju-websoft/JAPE) only contain 5 files: ent_ILLS, att_triples for each language and rel_triples for each language. Can you explain the process you use to get training_attrs files?

**Comment by 1049451037** [comment]
It is extracted from the initial dataset from http://ws.nju.edu.cn/jape/

**Comment by soummyaah** [comment]
Yes, I understand. Can you explain the extraction method? Particularly for the training_attrs files.

**Comment by 1049451037** [comment]
Oh, sorry. I find the training_attrs is the original data file from https://github.com/nju-websoft/JAPE/blob/master/data/dbp15k.tar.gz

I didn't modify it.
